guide/additional-info/async-await.md

```diff
@@ -4,17 +4,17 @@ If you aren't very familiar with ECMAScript 2017, you may not know about async/a
 
 ## How do Promises work?
 
-Before we can get into async/await, you should know what Promises are and how they work, because async/await is just a way to handle Promises. If you know what Promises are and how to deal with them, then you can skip this part. 
+Before we can get into async/await, you should know what Promises are and how they work because async/await is just a way to handle Promises. If you know what Promises are and how to deal with them, then you can skip this part. 
 
-Promises are a way to handle asynchronous tasks in Javascript; they are the newer alternative to callbacks. A Promise has a lot of similarities to a progress bar; Promises represent an ongoing process that has not yet finished. A good example for that is a request to a server (e.g discord.js sends requests to Discord's API).
+Promises are a way to handle asynchronous tasks in Javascript; they are the newer alternative to callbacks. A Promise has a lot of similarities to a progress bar; Promises represent an ongoing process that has not yet finished. A good example for that is a request to a server (e.g., discord.js sends requests to Discord's API).
```

Comment:
```diff
- Javascript
+ JavaScript```

You mix multiple with single in the same sentence - `Promises represent an ongoing process that has not yet finished.`  Something like `A Promise represents an ongoing process...` would be better.